src/test/scala/com/cognite/sdk/scala/v1/SequencesTest.scala

```diff
@@ -181,15 +181,33 @@ class SequencesTest extends SdkTestSpec with ReadBehaviours with WritableBehavio
     Some(client.sequences),
     sequencesToCreate,
     Seq(
-      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-1-1"))),
-      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-2-1")), dataSetId = Some(SetValue(testDataSet.id))),
+      // Column updates: modify the existing column
+      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-1-1")),
+        columns = Some(SequenceColumnsUpdate(
+          modify = Some(Seq(SequenceColumnModifyUpdate(
+            externalId = sequencesToCreate.head.columns.head.externalId.getOrElse(""),
+            update = SequenceColumnModify(name = Some(SetValue("you-are-string!!!")),
+            metadata = Some(SetValue(Map("test1" -> "0")))))))))),
+      // Column updates: add a new column and remove the existing column
+      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-2-1")), dataSetId = Some(SetValue(testDataSet.id)),
+        columns = Some(SequenceColumnsUpdate(add = Some(Seq(SequenceColumnCreate(name = Some("new-column-new-starts"),
+          externalId = s"${sequencesToCreate(2).columns.head.externalId.getOrElse("")}-2"))),
+          remove = Some(Seq(CogniteExternalId(sequencesToCreate(2).columns.head.externalId.getOrElse(""))))))),
       SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-3-1")), dataSetId = Some(SetNull()))
     ),
     (readSequences: Seq[Sequence], updatedSequences: Seq[Sequence]) => {
       assert(readSequences.size == updatedSequences.size)
       assert(updatedSequences.zip(readSequences).forall { case (updated, read) =>
         updated.name.value === s"${read.name.value}-1"
       })
+
+      updatedSequences.head.columns.head.metadata shouldBe Some(Map("test1" -> "0"))
+      updatedSequences.head.columns.head.name shouldBe Some("you-are-string!!!")
+
+      updatedSequences(1).columns.length shouldBe 1
+      updatedSequences(1).columns.head.name shouldBe Some("new-column-new-starts")
+      updatedSequences(1).columns.head.externalId shouldBe sequencesToCreate(2).columns.head.externalId.map(ext => s"$ext-2")
+
```

Comment:
```suggestion
      val firstSequenceFirstColumn = updatedSequences.headOption.map(_.columns.head)
      firstSequenceFirstColumn.flatMap(_.metadata) shouldBe Some(Map("test1" -> "0"))
      firstSequenceFirstColumn.flatMap(_.name) shouldBe Some("you-are-string!!!")

      val secondSequence = updatedSequences.lift(1).map(_.columns)
      secondSequence.map(_.size) shouldBe Some(1)
      val secondSequenceFirstColumn = secondSequence.map(_.head)
      secondSequenceFirstColumn.flatMap(_.name) shouldBe Some("new-column-new-starts")
      secondSequenceFirstColumn.flatMap(_.externalId) shouldBe sequencesToCreate.lift(2).flatMap(_.columns.head.externalId.map(ext => s"$ext-2"))

```

Reply:
just to avoid using .head and .(index) when accessing element in collections 😁

Reply:
this looks nicer but why? :p ty

Reply:
ok I see, to get it as an option ty